includes/wc-webhook-functions.php

```diff
@@ -19,9 +19,13 @@ function wc_webhook_execute_synchronous_queue() {
 		return;
 	}
 
+	add_filter( 'woocommerce_webhook_retry_enabled', '__return_false' );
```

Comment:
I'd be worried about what might happen if this is overridden by another filter. It might be better to check if the queue is synchronous when deciding whether or not an action can be retried?

Reply:
Seems that the code calling this changed in master -- so I just dropped the check.